pkg/flash/flash_linux.go

```diff
@@ -129,11 +155,12 @@ func (f *Flash) ReadAt(p []byte, off int64) (int, error) {
 func (f *Flash) writeAt(p []byte, off int64) (int, error) {
 	if err := f.spi.Transfer([]spidev.Transfer{
 		// Enable writing.
-		{Tx: []byte{op.WriteEnable}, CSChange: true},
+		{Tx: []byte{byte(op.WriteEnable)}, CSChange: true},
 		// Send the address.
-		{Tx: append([]byte{op.PageProgram}, f.prepareAddress(off)...)},
+		{Tx: append([]byte{byte(op.PageProgram)}, f.prepareAddress(off)...)},
```

Comment:
Make this less wordy:

```suggestion
		{Tx: append(op.PageProgram.Bytes(), f.prepareAddress(off)...)},
```